packages/mobile/src/screens/create-chat-blast-screen/ChatBlastSelectAudienceFields.tsx

```diff
@@ -70,7 +70,7 @@ const LabelWithCount = (props: {
 }
 
 const FollowersMessageField = () => {
-  const { data: user } = useGetCurrentUser()
+  const { data: user } = useGetCurrentUser({})
```

Comment:
could make useGetCurrentUser's default args = `{}` so this isn't needed

Reply:
Yes. I want to do this and will in a follow-up. It would mean updating the underlying hook as well, which caused a lint error in about 20 other call sites that aren't related to this work. So I didn't want to further inflate the PR with those changes.

Reply:
Oh oops, I read your suggestion too quickly. Yes the default arg would clean this up. But the argument doesn't need to exist at all if I update things as outlined above.